src/main/java/baseball/Ball.java

```diff
@@ -0,0 +1,26 @@
+package baseball;
+
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+import lombok.Setter;
+
+@Getter
+public class Ball {
+    private int num;
+    @Setter private int position;
+
+    public Ball(int num){
+        this.num=num;
+    }
+
+    @Override
+    public boolean equals(Object object){
```

Comment:
코딩 컨벤션을 지키면서 코드를 작성하도록 해주세요 :)
이클립스나 인텔리제이 같은 IDE에서는 보통 이런 기능들을 지원해주기 때문에 이를 활용해보면 좋을 것 같습니다 :)
```suggestion
    public boolean equals(Object object) {
```